Contributors/kunalgurjar

```diff
@@ -0,0 +1,2 @@
+Swapnil M Mane (@swapnilmmane)
+Kunal Gurjar (@kunalgurjar)
```

Comment:
It seems you deleted the swapnilmmane.md file, could you please restore it, thanks!